packages/webdriverio/src/commands/element/getCSSProperty.ts

```diff
@@ -1,6 +1,11 @@
 import cssShorthandProps from 'css-shorthand-properties'
 import { parseCSS } from '../../utils/index.js'
 
+type PseudoElement = {
+    target: '::before'|'::after',
+    browser: WebdriverIO.Browser | WebdriverIO.MultiRemoteBrowser
```

Comment:
Why should users pass in the `browser` instance? You can just use `this` or use the `getBrowserObject` to get the browser instance from an element

Reply:
I updated the code, browser is no longer in params, I get it via `getBrowserObject ` inside `getCSSProperty`. As for using `execute` on element it is not working.